Makefile

```diff
@@ -1,6 +1,7 @@
 build:
 	go build -o bin/kaigara cmd/kaigara/kaigara.go
 	go build -o bin/kaitail cmd/kaitail/kaitail.go
+	go build -o bin/kaisave cmd/kaisave/kaisave.go
```

Comment:
better to use the syntax
```
	go build -o bin/kaisave ./cmd/kaisave/
```
so we can potentially put more go files in the directory